### PR TITLE
Convert more-info-cover to TypeScript/LitElement

### PR DESCRIPTION
**Pull request by iantrich**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-cover.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      this._entityObj = new CoverEntity(this.hass, this.stateObj);
+    }
+  }
+
+  protected render(): TemplateResult {
+    if (!this._entityObj) {
+      return html``;
+    }
+
+    return html`<div class=${classMap(this._computeClassNames())}>
```
Comment (iantrich) [inline src/dialogs/more-info/controls/more-info-cover.ts:52]: I'm not sure how to convert this computeClassNames over

Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-cover.ts:52]: Looks like we don't need all those classes, it is either `max-height: 0px;` or `max-height: 280px;`

So make a `classMap` with 2 classes and the right checks <https://github.com/home-assistant/frontend/pull/7722/files#diff-dca629d25d790b59918244ca041907b2bc24c7b89de8f7346020e710a6191ad2R110-R121>

File: src/dialogs/more-info/controls/more-info-cover.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        <div class="current_position">
+          <ha-labeled-slider
+            .caption="[[localize('ui.card.cover.position')]]"
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-cover.ts:55]:
```suggestion
            .caption=${this.hass.localize("ui.card.cover.position")}
```

File: src/dialogs/more-info/controls/more-info-cover.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            pin=""
+            .value=${this.stateObj.attributes.current_position}
+            .disabled=${!this._entityObj.supportsSetPosition}
+            @on-change=${this._coverPositionSliderChanged}
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-cover.ts:59]:
```suggestion
            @change=${this._coverPositionSliderChanged}
```

File: src/dialogs/more-info/controls/more-info-cover.ts [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        </div>
+
+        <div class="tilt">
+          <ha-labeled-slider
+            .caption=${this.hass.localize("ui.card.cover.tilt_position")}
+            pin=""
+            extra=""
+            .value=${this.stateObj.attributes.current_tilt_position}
+            .disabled=${!this._entityObj.supportsSetTiltPosition}
+            @on-change=${this._coverTiltPositionSliderChanged}
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-cover.ts:70]:
```suggestion
            @change=${this._coverTiltPositionSliderChanged}
```

File: src/dialogs/more-info/controls/more-info-cover.ts [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+            .disabled=${!this._entityObj.supportsSetPosition}
+            @on-change=${this._coverPositionSliderChanged}
+          ></ha-labeled-slider>
+        </div>
+
+        <div class="tilt">
+          <ha-labeled-slider
+            .caption=${this.hass.localize("ui.card.cover.tilt_position")}
+            pin=""
+            extra=""
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-cover.ts:67]:
```suggestion
            pin
            extra
```

**Pull request closed without merging**